database/rrd.h

```diff
@@ -46,6 +46,8 @@ extern time_t rrdset_free_obsolete_time;
 #define RRDSET_MAGIC        "NETDATA RRD SET FILE V019"
 #define RRDDIMENSION_MAGIC  "NETDATA RRD DIMENSION FILE V019"
 
+#define STREAMING_PROTOCOL_CURRENT_VERSION (unsigned int)2
```

Comment:
I am being picky, I know :slightly_smiling_face: 
this is unsigned int and in the data structure defined as uint32_t

Reply:
According the definition of `bits/types.h` `uint32_t` is a ` unsigned int`, but I agree that is more readable to set as `uint32_t` here, so I changed in the last commit! :) 
You are not being "picky", you are doing the correct, but I confess sometimes I am not so rigorous, because I am thinking more in the ASM code instead of the code to be readable .